easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Binary'
+
+name = 'tecplot360ex'
+version = 'linux64'
+
+homepage = 'http://www.tecplot.com/'
```

Comment:
tecplot-360 ex is actually something different from tecplot-360 I believe, but the website is very unclear...